File: module-1/Week3_Review/exercise-final/dotnet/PetElevator.Tests/PetTests.cs
using Microsoft.VisualStudio.TestTools.UnitTesting;
using PetElevator.CRM;

namespace PetElevator.Tests
{
    [TestClass]
    public class PetTests
    {
        [TestMethod]
        public void ListVaccinations_Test()
        {
            Pet pet = new Pet("Fido", "Dog");

            pet.Vaccinations.Add("Rabies");
            pet.Vaccinations.Add("Distemper");
            pet.Vaccinations.Add("Parvo");

            string vaccString = pet.ListVaccinations();

            Assert.AreEqual("Rabies, Distemper, Parvo", vaccString);
        }
    }
}

File: module-1/Week3_Review/exercise-final/java/src/test/java/com/techelevator/crm/PetTests.java
package com.techelevator.crm;

import org.junit.Assert;
import org.junit.Test;

public class PetTests {

    @Test
    public void listVaccinations() {
        Pet pet = new Pet("Fido", "Dog");
        pet.getVaccinations().add("Rabies");
        pet.getVaccinations().add("Distemper");
        pet.getVaccinations().add("Parvo");

        Assert.assertEquals("List Vaccinations is not returning the vaccinations in the correct format.",
                "Rabies, Distemper, Parvo",
                pet.listVaccinations());
    }
}
